src/components/landing/search-result.js

```diff
@@ -74,6 +74,8 @@ const SearchResult = ({ searchResult }) => {
                   The origin of your name is {searchResult.origin}
                 </p>
               </div>
+
+              <Addmoredetail />
```

Comment:
You can add a simple and small size link here. We may not need the whole new component for this.